lib/task-data/tasks/set-boot-disk.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Set Node Diskboot',
+    injectableName: 'Task.Obm.Node.DiskBoot',
+    implementsTask: 'Task.Base.Obm.Node',
```

Comment:
Refer to `setPxeBoot` (https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/set-boot-pxe.js#L9) , please specify the `schemaRef`